src/main/proto/eventtemplates/display_template.proto

```diff
@@ -28,17 +28,23 @@ message Display {
     description: "An Event Template for display ad events."
```

Comment:
The `name` in the `event_template` annotation needs to be lowercase, as it **must** match the field name in the corresponding event message.

This comment is actually for line 27, and also applies to the video template.

Reply:
thank you. you mean line 26 right? That is the `name` field. 27 is `display_name`